openmetadata-ui/src/main/resources/ui/playwright/support/entity/ingestion/ServiceBaseClass.ts

```diff
@@ -170,7 +170,7 @@ class ServiceBaseClass {
 
     // Header available once page loads
     await page.waitForSelector('[data-testid="data-assets-header"]');
-    await page.getByTestId('loader').waitFor({ state: 'detached' });
+    await page.getByTestId('loader').first().waitFor({ state: 'detached' });
```

Comment:
I think following would be a better option here instead of first() to make sure we are waiting for the exact loader
```
await page
      .getByTestId('table-container')
      .getByTestId('loader')
      .waitFor({ state: 'detached' });
```